src/components/layout/yearInReview/2023/collaborations.tsx

```diff
@@ -0,0 +1,177 @@
+                    id: 'page.year-in-review.2023.section.collaborations.1.heading',
+                    defaultMessage:
+                      'Collaboration for <b>EFFECTIVE REDIRECTION</b>',
+                  })}
+                  smallOnMobile={true}
+                  stackEntries
+                />
+              </div>
+              <FormattedMessage
+                id='page.year-in-review.2023.section.collaborations.1.content'
+                defaultMessage="This year, we partnered with <no-localization>AAC</no-localization> by winding down their skilled volunteer board's activities and guiding those eager to volunteer or seeking volunteer positions towards <no-localization>VH Playground</no-localization>. Likewise, <no-localization>Playground</no-localization> began to advocate <no-localization>AAC's</no-localization> career opportunities within our volunteer community for individuals aspiring to pursue a more career-oriented role."
+                values={{
+                  b: (chunks) => <b>{chunks}</b>,
+                  break: <br />,
+                }}
+              />
+            </div>
+            <div className='bg-[#F1F1F1] p-10 flex items-center justify-center max-w-xl mx-auto w-full'>
+              <CustomImage alt='Animal Advocacy Careers' src={ACCLogo} />
+            </div>
+          </div>
+          <div className='flex flex-col lg:grid lg:grid-cols-3 gap-10'>
+            <div className='text-lg text-left lg:col-span-2'>
+              <div className='flex items-center mb-5'>
+                <div className='bg-yellow-orange flex items-center justify-center text-3xl md:text-7xl text-white font-bold font-mono w-10 md:w-20 h-10 md:h-20 p-3 mr-3 md:mr-10'>
+                  2
+                </div>
+                <SectionHeader
+                  header={intl.formatMessage({
+                    id: 'page.year-in-review.2023.section.collaborations.2.heading',
+                    defaultMessage:
+                      'Partnership for <b>ADVANCED CONVERSATIONS</b>',
+                  })}
+                  smallOnMobile={true}
+                  stackEntries
+                />
+              </div>
+              <FormattedMessage
+                id='page.year-in-review.2023.section.collaborations.2.content'
+                defaultMessage='We partnered with <no-localization>Sentient Media’s Writers’ Collective</no-localization> to train and mentor new writers focused on topics like animal welfare, plant-based diets, social justice, and the environment. By guiding almost 200+ writer applicants to their program, we support enriching these important conversations.'
+                values={{
+                  b: (chunks) => <b>{chunks}</b>,
+                  break: <br />,
+                }}
+              />
+            </div>
+            <div className='bg-[#F1F1F1] p-10 flex items-center justify-center max-w-xl mx-auto w-full'>
+              <svg
```

Comment:
Please move the svg to a separate file to keep this component small. The file will be inlined by the bundler.